code/game/objects/items/tcg/tcg.dm

```diff
@@ -168,6 +168,25 @@ GLOBAL_LIST_EMPTY(cached_cards)
 	. = ..()
 	transform = matrix(0.4,0,0,0,0.4,0)
 
+/obj/item/storage/card_binder
+	name = "card binder"
+	desc = "The perfect way to keep your collection of cards safe and valuable."
+	icon = 'icons/obj/tcg.dmi'
```

Comment:
I really recommend not having misc icons and the first series of cards in the same file.
Doesn't have to be done right now but needs to be done eventually.

Reply:
Yeah the more I think about this the more a misc TCG file for cardbacks and such feels like a good idea. I'll sort that out tomorrow and include it in this PR.